addons/l10n_it_edi_sdi_coop/models/ir_mail_server.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class FetchmailServer(models.Model):
+    _name = 'fetchmail.server'
+    _inherit = 'fetchmail.server'
+
+    def use_pec(self):
```

Comment:
seems overkill

Reply:
You told me to do it this way :D https://github.com/odoo/odoo/pull/67923#discussion_r595062239
We said we deactivate the PEC mail when sdicoop is installed.

Reply:
yes, but I wonder if the post_init_hook would not be the simplest.